lib/workos/group.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+# typed: true
+
+module WorkOS
+  # The Group class provides a lightweight wrapper around
+  # a WorkOS Group resource. This class is not meant to be instantiated
+  # in user space, and is instantiated internally but exposed.
+  class Group
```

Comment:
How are we naming this in other SDKs? I feel this should be named `DirectoryGroup` to a.) be more specific and consistent with our API b.) clear up the `Group` name in case we ever name something in our API that.

Reply:
That's a good point - I think it's a bit all over the map, but I like the more descriptive naming. I'll update `Group` and `User` to `DirectoryGroup` and `DirectoryUser` respectively, if that sounds good.

Reply:
That sounds great.